### DISCUSSION TITLE
TO DO list -  Qibo site

### DISCUSSION THREAD

**Discussion by MatteoRobbiati** [qibogang/qibogang.github.io#31] [General]
##### About `README.md` 

- [x] re-write the installation procedure because at the moment we don't use the sub-modules;
- [x] explain how to install `nodejs` (required by yarn);
- [ ] add the `yarn next export` instruction;
- [x] correct link to `INSTALL.md`.

#####  About the page

- [x] make the carousel more visible;
- [x] too much space between carousel and getting started section in the home-page;
- [x] move the partners section on top: use the official logos colors and align them to the vertical center;
- [x] add links to partners' pages onclick over the logos;
- [x] smaller images into tutorials' cards;
- [x] choose a general font for the text and use it into furo documentation;
- [x] navbar with the same dimension of qibocal reports one;
- [ ] change the home-page background because it doesn't fit high dimension monitors;
- [x] name of the home-page must be: `Qibo | Home Page`;
- [ ] clean all pages logs once the console is launched using the inspect tool;
- [ ] deploy both `latest` and `stable` pages;
- [ ] remove name, version and documentation sentence from the documentation;
- [ ] add qibocal and qibolab sentence into the documentation;
- [ ] add qibocal to `backends.png` together with qibolab;
- [ ] add code copy button into the tutorials (for easily copying the code blocks);
- [x] use the padding to make the tutorial fitting with the navbar;
- [x] in the Home Page "Getting started" section: we want the two buttons to be closer together;
- [x] change documentation draw at the end of the home page: it really looks like the Bible now.

I think that's all, let me know is something is missing.

**Comment by alecandido** [comment]
Try to gather them in issues, and add them to the project https://github.com/orgs/qibogang/projects/1

**Reply by MatteoRobbiati** [reply]
Yes, this is only a "note book", useful for me especially today and tomorrow.

**Reply by alecandido** [reply]
This is perfect, it's much better than a piece of paper, or a note in a laptop.

I was just inviting you to create issues before starting working on them :)
